### PR TITLE
Unify usage `setTimeout()` method

### PR DESCRIPTION
**Pull request by leon-win**
### Description

This PR brings all uses of the `setTimeout()` method to a single form:
- without unnecessay `Window.`,
- with `()`.

### Motivation

Content unification.

### Additional details

Also this PR removes some unused redirects.
This needs to be checked, but as far as I understand, these redirects are no longer needed.

### Related issues and pull requests

Relates to #6938

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -2241,7 +2241,6 @@
 /en-US/docs/DOM/window.self	/en-US/docs/Web/API/Window/self
 /en-US/docs/DOM/window.setImmediate	/en-US/docs/Web/API/Window/setImmediate
 /en-US/docs/DOM/window.setInterval	/en-US/docs/Web/API/setInterval
-/en-US/docs/DOM/window.setTimeout	/en-US/docs/Web/API/setTimeout
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:2244]: This file shouldn't be edited!

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Do not change _redirects.txt: the redirections should stay. They are for external links linking to MDN.

**Comment by leon-win** [comment]
> Do not change _redirects.txt: the redirections should stay. They are for external links linking to MDN.

I have reverted the changes to this file in taht PR.

But I think that in general the issue of preserving redirects unused inside `mdn/content` should be discussed. Because if you do not clean it up, this leads to their uncontrolled growth.

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (leon-win) [reply files/en-us/_redirects.txt]: Reverted :ok_hand:

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**